learn/os_setup/install_gfortran.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: book
+title: Installing GFortran
+permalink: /learn/os_setup/install_gfortran
+---
+
+GFortran is the name of the [GNU Fortran project](https://gcc.gnu.org/fortran/). The main [wiki](https://gcc.gnu.org/wiki/GFortran) page offers many helpful and informational links about GFortran, as well as Fortran in general. In this guide, the installation process for GFortran on Windows, Linux, and MacOS is presented in a beginner-friendly format based on information from [GFortranBinaries](https://gcc.gnu.org/wiki/GFortranBinaries).
+
+## Windows
+
+In Windows, there are two main *vendors* that offer builds of GFortran, namely the [MinGW](http://www.mingw.org/) and the [Cygwin](https://cygwin.com/index.html). Directly derived from MinGW is the [Mingw-w64](http://mingw-w64.org/) project that offers 64-bit executables for many platforms. 
```

Comment:
Hm, I would not call Cygwin and MinGW "vendors" - not sure what to call them instead. Perhaps "user environments"? Sounds a bit lame, though.

Reply:
I would refer to them as different ports of the GNU compiler collection (and related tooling) to Windows; Cygwin aims to provide a large potion of POSIX functionality through its Cygwin environment, whereas MinGW instead aims for native Windows compatibility (based off Microsoft libraries).

Reply:
@arjenmarkus following @LKedward advice I just removed this specific part altogether, in order to avoid this unnecessary confusion at a beginner's tutorial.